metamorph/processor.go

```diff
@@ -225,62 +215,50 @@ func (p *Processor) processMinedCallbacks() {
 	}()
 }
 
-func (p *Processor) processStatusUpdates() {
+func (p *Processor) CheckAndUpdate(statusUpdatesMap *map[chainhash.Hash]store.UpdateStatus) {
```

Comment:
I think the map does not need to be passed as a pointer becauce [Map types are reference types, like pointers or slices](https://go.dev/blog/maps)

Reply:
<img width="554" alt="Screenshot 2024-03-08 at 11 01 12" src="https://github.com/bitcoin-sv/arc/assets/139438093/78443d81-d5e4-4c65-8902-58b58940df2c">